src/workloads/execution/BackgroundTTLDeletions.yml

```diff
@@ -0,0 +1,187 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+Description: |
+  This workload tests the impact of background TTL deletions in a heavily modified collection
```

Comment:
Could you add in a reference to the legacy workload and how this relates? It looks pretty close to a straight port, with only minor differences.

Reply:
Done in the latest commit